lib/no-code/code-generator.ts

```
@@ -0,0 +1,462 @@
+      const name = this.getControlValue(node, "name");
+      const visibility = this.getControlValue(node, "visibility") || "public";
+      const value = this.getControlValue(node, "value");
+
+      if (!name) continue;
+
+      switch (node.label) {
+        case "Uint Variable":
+          const declaration =
+            value && value !== "0"
+              ? `uint256 ${visibility} ${name} = ${value};`
+              : `uint256 ${visibility} ${name};`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)